megalinter/descriptors/kubernetes.megalinter-descriptor.yml

```diff
@@ -113,6 +113,9 @@ linters:
         - libc6-compat
         - libstdc++
       dockerfile:
-        - |-
+        - |
+          # renovate: datasource=github-tags depName=kubescape/kubescape
+          ARG KUBESCAPE_VERSION=3.0.14
```

Comment:
I'm not completely sure it will work, as the tags in the repo contain a v before. We can try it, and see if it complains (in the dependency dashboard issue), as their semver coercion is pretty good.

(And you added the v below).

Also, there's a datasource that is github-release-attachments, but I don't know how to use it, https://docs.renovatebot.com/modules/datasource/github-release-attachments/